app/femr/ui/views/admin/configure/manage.scala.html

```diff
@@ -13,13 +13,14 @@
             <tr>
                 <th>Feature</th>
                 <th>Toggle</th>
+                <th>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Description</th>
             </tr>
 
             @for((key, valyew) <- viewModel.getSettings) {
                 <tr>
                     <td class="name">@key</td>
                     <td class="isActive">
-                        <input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} />
+                        <input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} >
                     </td>
```

Comment:
Maybe this was taken out by accident, but the closing slash makes that input field valid XHTML. This is not the end of the world, but could you add that back in? Line 23 should read:

`<input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} />`